### PR TITLE
feat(card, file): add highlight label - FRONT-3276

### PR DESCRIPTION
**Pull request by emeryro**

### CODE REVIEW

File: src/implementations/twig/components/card/__snapshots__/card.test.js.snap [hunk 1]
```diff
@@ -129,6 +151,28 @@ exports[`Card Default renders correctly with extra attributes 1`] = `
...
+            class="ecl-label ecl-label--highlight ecl-card__label"
+          >
+            highlighted
+          </span>
+        </li>
+        <li
+          class="ecl-card__label-item"
+        >
+          <span
+            class="ecl-label ecl-label--medium ecl-card__label"
```
Comment (papegaill) [inline src/implementations/twig/components/card/__snapshots__/card.test.js.snap:170]: on playground this label is highlight, blue color? normal? https://ecl-preview-2232--europa-component-library.netlify.app/playground/ec/?path=/story/components-card--card

Comment (papegaill) [reply src/implementations/twig/components/card/__snapshots__/card.test.js.snap:170]: for me as an example it is better to use the "high important" variant for this label, as in figma `ecl-label--high`

**Review by papegaill — CHANGES_REQUESTED** [review]

File: src/implementations/twig/components/file/__snapshots__/file.test.js.snap [hunk 2]
```diff
@@ -16,6 +16,15 @@ exports[`File Thumbnail renders correctly 1`] = `
...
         >
+          <div
+            class="ecl-file__label"
+          >
+            <span
+              class="ecl-label ecl-label--highlight"
+            >
+              highlighted
+            </span>
+          </div>
```
Comment (papegaill) [inline src/implementations/twig/components/file/__snapshots__/file.test.js.snap:27]: for the file, should we also have a yellow “high important” label as in figma?

File: src/implementations/twig/components/card/__snapshots__/card.test.js.snap (hunk 1 above)
Comment (emeryro) [reply src/implementations/twig/components/card/__snapshots__/card.test.js.snap:170]: yes I noticed this, but I didn't want to over-complicate the controls. They only offers the possibility to change the label, so the color is fixed on the example

File: src/implementations/twig/components/file/__snapshots__/file.test.js.snap (hunk 2 above)
Comment (emeryro) [reply src/implementations/twig/components/file/__snapshots__/file.test.js.snap:27]: I used this wiki page as a reference: https://citnet.tech.ec.europa.eu/CITnet/confluence/pages/viewpage.action?pageId=948443705
But in any case, the twig file expect a label component structure, so it is a matter of data to use one variant or another

**Pull request merged**